src/TConsole.cpp

```diff
@@ -247,7 +247,8 @@ void TConsole::Command_Help(const std::string&, const std::vector<std::string>&
         lua [state id]          switches to lua, optionally into a specific state id's lua
         settings [command]      sets or gets settings for the server, run `settings help` for more info
         status                  how the server is doing and what it's up to
-        clear                   clears the console window)";
+        clear                   clears the console window
+        version                 return current version to console)";
```

Comment:
Would be better if the wording here more closely matched the other help strings, like "displays this help", perhaps like:
```
        version                 displays the version of this server)";
```